src/locales/en.json

```diff
@@ -641,21 +641,21 @@
   "sarStakeMore": {
     "stakeMore": "Stake More",
     "choosePosition": "Choose a Position",
-    "confirmDescription": "Your Apr will start from 0 and increase the amount you take from reward pool by time",
+    "confirmDescription": "This adds {{ symbol }} to your SAR NFT position. Your original balance continues from the same APR. The APR of your newly added tokens starts from 0% and changes based on the length of stake. Your NFT will show the average.",
```

Comment:
as you have changed verbiage here with new `{{ symbol }}` variable, please do this for all languages. also please check other instances of same issue